scripts/get_counts.py

```diff
@@ -433,6 +445,17 @@ def MergeDevData(dest_count_dir, ngram_order):
 num_train_sets = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
+if (args.max_memory != '' and args.dump_counts_parallel == 'true'):
+  sub_memory = ParseInt(args.max_memory)/num_train_sets
```

Comment:
you need to convert int to float before dividing. if args.max_memory == 1 and num_train_sets = 2, the result would be zero.